### PR TITLE
Improve ja workflow page

### PR DESCRIPTION
**Pull request by bufferings**
# Description

Improved Japanese translation page of workflow

* Fix header to use the same definition as English ver
* Remove unnecessary italic markdown
* Translate English sentence
* Fix invalid link
* Fix invalid yaml
* Fix video parts

# Reasons

I read the Japanese page and found there were several small things to be improved.

### CODE REVIEW

File: jekyll/_cci2_ja/workflows.md [hunk 1]
```diff
@@ -655,8 +653,12 @@ GitHub で [Settings (設定)] > [Branches (ブランチ)] に移動し、保護
 
 ## ビデオ: ワークフローに複数のジョブを設定する
 {: #video-configure-multiple-jobs-with-workflows }
-{:.no_toc} <iframe width="560" height="315" src="https://www.youtube.com/embed/3V84yEz6HwA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen mark="crwd-mark"></iframe>
+{:.no_toc}
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/3V84yEz6HwA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (bufferings) [inline jekyll/_cci2_ja/workflows.md:658]: When I added line break, I found English version doesn't have `mark="crwd-mark"` attribute. So I removed it.

https://github.com/circleci/circleci-docs/blob/c69d09ebf264d2432b7d1f0946dbf9a51a484ab3/jekyll/_cci2/workflows.md?plain=1#L671

File: jekyll/_cci2_ja/workflows.md [hunk 2]
```diff
@@ -655,8 +653,12 @@ GitHub で [Settings (設定)] > [Branches (ブランチ)] に移動し、保護
...
 
 ### ビデオ: 自動的にテストおよびデプロイを行うようビルドのスケジュールを設定する
 {: #video-how-to-schedule-your-builds-to-test-and-deploy-automatically }
-{:.no_toc} <iframe width="560" height="315" src="https://www.youtube.com/embed/FCiMD6Gq34M" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen mark="crwd-mark"></iframe>
```
Comment (bufferings) [inline jekyll/_cci2_ja/workflows.md:662]: Same on this. I removed `mark="crwd-mark"`

**Comment by bufferings** [comment]
Thank you so much for your comment, @devinmarieb 

I was told that Ja translation uses some tools and not using GitHub directly.
So it's no problem to close this PR. Thank you for your consideration :) 

@Mayuko-Sa Yoroshiku Onegai Shimasu!

**Comment by Mayuko-Sa** [comment]
Thank you @bufferings for the heads-up.
I'll deal with your contribution in the internal translation management system. The fix will be published within a few weeks .

**Pull request closed without merging**